.github/workflows/build-and-deploy.yml

```diff
@@ -30,12 +29,12 @@ jobs:
 
       - name: Deploy
         # Only deploy for commits on 'master'
-        if: github.event_name == 'push' && github.ref == 'refs/heads/master'
-        uses: JamesIves/github-pages-deploy-action@releases/v3
+        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
+        uses: JamesIves/github-pages-deploy-action@v4
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           BRANCH: gh-pages
           FOLDER: dist
-          GIT_CONFIG_NAME: GitHub Actions
+          GIT_CONFIG_NAME: actions-user
```

Comment:
Also doesn't do anything since what is displayed on GitHub, just isn't a bad idea. It inherits from the email first.